crates/rapier2d-f64/Cargo.toml

```diff
@@ -74,15 +74,15 @@ approx = "0.5"
 rayon = { version = "1", optional = true }
 crossbeam = "0.8"
 arrayvec = "0.7"
-bit-vec = "0.7"
+bit-vec = "0.8"
 rustc-hash = "2"
 serde = { version = "1", features = ["derive"], optional = true }
-downcast-rs = "1.2"
+downcast-rs = "2"
```

Comment:
Thanks! To keep track I'm noting that parry upgraded downcast to version 2 in https://github.com/dimforge/parry/pull/305 ; we'll want to update rapier to the future parry version to avoid duplicate dependencies